doc.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Documentation</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="doc.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div class="navbar">
+      <h5 class="logo">BlossomUI</h5>
+      <!-- <img class="logo" src="images/logo.png" alt="logo of website  BlossomUI" /> -->
+      <nav>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./doc.html">Documentation</a></li>
+          <li><a href="http://#">Github</a></li>
+        </ul>
+      </nav>
+      <a class="ham-burger" onclick="sideBarShow()">
+        <i class="fa fa-bars"></i>
+      </a>
+      <sidebar class="navsidebar">
+        <span> <a href="./index.html">Home</a> </span>
+        <span> <a href="./doc.html">Documentation</a> </span>
+        <span> <a href="#">Github</a> </span>
+      </sidebar>
+    </div>
+    <div class="componentSidebar">
+      <div class="sidebar-main">
+        <h3>Installation Guid</h3>
+        <a href="#installation-content" class="install-guid">Installation</a>
+        <hr />
+        <h3>Components</h3>
+        <a href="">Alert</a>
+        <a href="">Avatar</a>
+        <a href="">Badge</a>
+        <a href="">Button</a>
+        <a href="">Card</a>
+        <a href="">Image</a>
+        <a href="">Input</a>
+        <a href="">Text Utilities</a>
+        <a href="">Lists</a>
+        <a href="">Navigation</a>
+        <a href="">Modal</a>
+        <a href="">Rating </a>
+        <a href="">Toast</a>
+        <a href="">Responsive Grid</a>
+        <a href="">Slider</a>
+      </div>
+    </div>
+    <div class="main-content">
+      <div class="installation-content">
+        <div>
+          <h1>Install Blossom UI</h1>
+          <p>
+            Copy and paste the code in the head tag of your html document. So
+            that you can style your components by just adding class names to
+            your html elements.
+          </p>
+        </div>
+
+        <div class="code_link">
+          <code>
+            <iframe
```

Comment:
Here, the height of this container can be adjusted as it's making it scroll, which is not a very good UI.
![Screenshot 2022-02-01 at 21 46 36](https://user-images.githubusercontent.com/62737267/152007169-74806524-19cb-49ef-95ba-66d04f55594c.png)